data.sql

```diff
@@ -43,3 +43,41 @@ UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon'); 
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom'); 
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon'); 
+
+--DAY 4
+INSERT INTO vets(name, age, date_of_graduation)
+VALUES
+	('Vet William Tatcher', '45',  date '2000-04-23'),
+	('Vet Maisy Smith', '26',  date '2019-01-17'),
+	('Vet Stephanie Mendez', '64',  date '1981-05-04'),
+	('Vet Jack Harkness', '38',  date '2008-06-08');
+
+INSERT INTO specialization(vets_id, species_id)
+VALUES
+	('1', '1'),
+	('3', '1'),
+	('3', '2'),
+	('4', '2');
+
+INSERT INTO visits(animals_id, vets_id, date_of_visit)
+VALUES
+	('1', '1', date '2020-05-24'),
+	('1', '3', date '2020-07-22'),
+	('2', '4', date '2021-02-02'),
+	('3', '2', date '2020-01-05'),
+	('3', '2', date '2020-03-08'),
+	('3', '2', date '2020-05-14'),
+	('4', '3', date '2021-05-04'),
+	('5', '4', date '2021-02-24'),
+	('6', '2', date '2019-12-21'),
+	('6', '1', date '2020-08-10'),
+	('6', '2', date '2021-04-07'),
+	('7', '3', date '2019-09-29'),
+	('8', '4', date '2020-10-03'),
+	('8', '4', date '2020-11-04'),
+	('9', '2', date '2019-01-24'),
+	('9', '2', date '2019-05-15'),
+	('9', '2', date '2020-02-27'),
+	('9', '2', date '2020-08-03'),
+	('10', '3', date '2020-05-24'),
+	('10', '1', date '2021-01-11');	
```

Comment:
Propper data was introduced into the new tables 😁

Reply:
Thank you very much